tslint.json

```diff
@@ -1,33 +1,21 @@
 {
   "rulesDirectory": ["node_modules/codelyzer"],
   "rules": {
-    "no-unused-variable": true,
     "arrow-return-shorthand": true,
     "callable-types": true,
     "class-name": true,
-    "comment-format": [true, "check-space"],
-    "curly": true,
     "deprecation": {
       "severity": "warn"
     },
-    "eofline": true,
     "forin": true,
     "import-blacklist": [true, "rxjs/Rx"],
-    "import-spacing": true,
-    "indent": [true, "spaces"],
     "interface-over-type-literal": false,
     "label-position": true,
-    "max-line-length": [true, 140],
     "member-access": false,
     "member-ordering": [
       true,
       {
-        "order": [
-          "static-field",
-          "instance-field",
-          "static-method",
-          "instance-method"
-        ]
+        "order": ["static-field", "instance-field", "static-method", "instance-method"]
```

Comment:
I assume this is still 140 length ;) ?

Reply:
No, that was removed here: https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/3124/files/328d8e2ba38955ea7deb64196695a533c7793226#diff-ace19bd0c04529e685320269e3c05de9L20 :)
We leave formatting to prettier, which (still) has `80` set here: https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/develop/.prettierrc#L2 ;)